examples/bodymovin-animation.amp.html

```diff
@@ -74,6 +74,36 @@ <h2>Example of a responsive layout animation</h2>
   <amp-bodymovin-animation renderer="html" layout="responsive" width="200" height="200" src="https://nainar.github.io/loader.json" loop="true">
   </amp-bodymovin-animation>
 
+  <h1>Canvas renderer</h1>
```

Comment:
I added examples, so that the functionality can be manually verified. I looked for existing tests for an example, as I wanted to add some, but it seems that all tests are "TODO": https://github.com/ampproject/amphtml/blob/master/extensions/amp-bodymovin-animation/0.1/test/integration/test-amp-bodymovin-animation.js.

Reply:
This is good, thank you! One thing to confirm: is 5.0 backwards compatible with 4.0 JSON descriptors?

Reply:
To the best of my knowledge, the only reason for a major version bump was the change of the name from Bodymovin to Lottie. All animations I worked with (some of which were very complex) render correctly with the new version. See https://github.com/airbnb/lottie-web/blob/master/CHANGELOG.md#v-500.

Reply:
Awesome, thanks!